.opspec/release/to-ghcr/op.yml

```diff
@@ -0,0 +1,51 @@
+name: to-docker
+description: releases opctl to [docker hub](https://hub.docker.com/r/opctl/opctl/)
+inputs:
+  encodedGithubAuth:
+    string:
+      description: github auth encoded as base64
+  version:
+    string:
+      constraints: { format: semver }
+      description: version being published
+  dockerSocket:
+    socket:
+      description: docker socket
+    default: /var/run/docker.sock
```

Comment:
```suggestion
```

Reply:
presumably the last 3 comments were all supposed to be single suggestion - I got rid of the unused param